packages/core/src/components/account/orders/MyAccountOrderDetails/MyAccountStatusCard/styles.scss

```
@@ -0,0 +1,218 @@
+[data-fs-order-status-card] {
+  // --------------------------------------------------------
+  // Design Tokens for Order Status Card
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-order-status-card-gap                     : var(--fs-spacing-1);
+  --fs-order-status-card-padding                 : var(--fs-spacing-3) var(--fs-spacing-2);
+  --fs-order-status-card-background-color        : var(--fs-color-neutral-1);
```

just to keep the pattern
```suggestion
  --fs-order-status-card-bkg-color        : var(--fs-color-neutral-1);
```